pom.xml

```diff
@@ -159,7 +159,7 @@ See the Apache License Version 2.0 for the specific language governing permissio
       <dependency>
         <groupId>cglib</groupId>
         <artifactId>cglib</artifactId>
-        <version>3.1</version>
+        <version>3.2</version>
```

Comment:
FYI the version on central appears to be 3.2.0: http://search.maven.org/#artifactdetails%7Ccglib%7Ccglib%7C3.2.0%7Cjar